proto/rill/admin/v1/api.proto

```diff
@@ -45,6 +45,24 @@ service AdminService {
     };
   }
 
+  // UpdateOrganizationBillingPlan updates the billing plan for the organization
+  rpc UpdateOrganizationBillingPlan(UpdateOrganizationBillingPlanRequest) returns (UpdateOrganizationBillingPlanResponse) {
+    option (google.api.http) = {
+      patch: "/v1/organizations/{org_name}/billing/plan",
+      body: "*"
+    };
+  }
+
+  // GetOrganizationBillingSubscription lists the subscription for the organization
+  rpc GetOrganizationBillingSubscription(GetOrganizationBillingSubscriptionRequest) returns (GetOrganizationBillingSubscriptionResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{org_name}/billing/subscriptions"};
+  }
+
+  // DeleteOrganizationSubscription deletes the given subscription for the organization
+  rpc DeleteOrganizationBillingSubscription(DeleteOrganizationBillingSubscriptionRequest) returns (DeleteOrganizationBillingSubscriptionResponse) {
+    option (google.api.http) = {delete: "/v1/organizations/{org_name}/billing/subscriptions/{subscription_id}"};
+  }
```

Comment:
Maybe we already discussed this – but why is it we need the ability to delete a subscription? Won't it put the org into a weird state? If you want to stop a paid subscription, shouldn't you just call `UpdateOrganizationBillingPlan` to change to a free plan?

Reply:
Yeah my original intention was to allow for fix for issues that could arise from manual assignment, also don't see any free plan in the mocks, I just see Starter plan which has 30 days trial period and then its all paid. In that case as well if no subscprition then org should be hibernated, not sure but its a separate issue. Anyways removed.